client/core/trade.go

```diff
@@ -1584,6 +1584,21 @@ func (c *Core) swapMatchGroup(t *trackedTrade, matches []*matchTracker, errs *er
 		errs.add("not broadcasting swap while DEX %s connection is down (could be revoked)", t.dc.acct.host)
 		return
 	}
+
+	// Use a higher swap fee rate if a local estimate is higher than the
+	// prescribed rate, but not higher than the funded (max) rate.
+	var freshRate uint64
+	if r, ok := t.wallets.fromWallet.feeRater(); ok {
+		freshRate, _ = r.FeeRate()
+	}
+	if freshRate == 0 { // either not a FeeRater, or FeeRate failed
+		freshRate = t.dc.bestBookFeeSuggestion(fromAsset.ID)
+	}
```

Comment:
I had trialed a new `cacheSwapFeeSuggestion` method, but decided to keep it simpler like this. It didn't add much to have it cached like the redeem fee.  It's guaranteed to be current this way too.